### PR TITLE
Use yapf for consistent formatting

### PR DESCRIPTION
**Pull request by nicksellen**
You can run it like this:

```
yapf -i -r -e 'foodsaving/*/migrations'  foodsaving
```

... and there is an intellij plugin.

### CODE REVIEW

**Comment by nicksellen** [comment]
Sometimes it makes multiline things go onto one line, but because I configured `split_arguments_when_comma_terminated` you can make it split to multiple lines by having a trailing comma.

**Comment by nicksellen** [comment]
Obviously, this cannot be merged, this is just for seeing whether we like it.

**Comment by tiltec** [comment]
Yay for autoformatting :)

File: foodsaving/conversations/api.py [hunk 1]
```diff
@@ -82,20 +74,16 @@ def email_notifications(self, request, pk=None):
         return Response(serializer.data)
 
 
-class ConversationMessageViewSet(
-    mixins.CreateModelMixin,
-    mixins.ListModelMixin,
-    GenericViewSet
-):
+class ConversationMessageViewSet(mixins.CreateModelMixin, mixins.ListModelMixin, GenericViewSet):
```
Comment (tiltec) [inline foodsaving/conversations/api.py]: I think we should go for the multiline version, putting a comma after the last entry.

**Comment by tiltec** [comment]
Maybe now is a good time to integrate yapf?

**Comment by nicksellen** [comment]
Sounds good. Might also be able to just use the _chromium_ preset, I used it for something else, and it seemed to fit what we like, didn't check in detail though.

Might need a totally branch on-top of current master. Or maybe just formatting master, then rebasing could work. Who knows.

**Comment by nicksellen** [comment]
Ok, I reworked this now on-top of master, and included `./yapf.py` script to run it (with our excludes).

In the future if https://github.com/google/yapf/pull/543 gets merged we can use a yapf ignore file instead.

So, the plan would be:
- merge this to master
- cherry pick the "add yapf config" commit into any PR branch you might want to merge in the future
- run `./yapf.py` on your branch
- hopefully when you come to merge it'll all work nicely :)

Until we have gone through all the active branches like this, we should not manually reformat too much, or change the yapf config... keeps the pain down a bit. After that maybe some cleanup phases to (for example add lots of trailing commas).

**Pull request merged**